src/secrets.cr

```diff
@@ -33,7 +33,12 @@ class Secrets
     data = data[IV_SIZE..]
     @aes = AES.new(key, iv, AES_BITS)
     decrypted = String.new(@aes.decrypt(data))
-    yaml = YAML.parse(decrypted)
+    begin
+      yaml = YAML.parse(decrypted)
+    rescue ex : YAML::ParseException
+      puts "Invalid YAML: #{ex.message}"
+      raise "YAML::ParseException: This could be due to an invalid key, bad encoding on the key or secrets file, or just bad yaml!"
```

Comment:
I believe these two strings should be switched.
The `puts` should be the more user-friendly message, but the original exception should be propagated up as it may contain hints as to what the issue is, for example, in the case of simply a bad YAML encoding, somehow.